src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,173 @@
+
+#### <a name="overviewV4"></a>Overview of Version 4.0
+
+Version 4.0 of the Requests API offers features like passive approval, Data retention, rigth to be forgotten.
+
+It works only with the new [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+### <a name="regionalavailability"></a>Regional Availability
+#### <a name="swagger"></a>With swagger
+For US production
+```
+https://sea-raas.concurasp.com
+```
+For EMEA production
+```
+https://par-raas.concurasp.com/
+```
+For China production
+```
+https://raas.service.cnqr.io/
+```
+#### <a name="curl"></a>With curl
+For US production
+```
+curl -X POST --H 'Content-Type: application/json' --H 'Accept: application/json' --header "Authorization: Bearer {YOUR ACCESS TOKEN}" -d { "name": "Your action name" } https://us.api.concursolutions.com/travelrequest/v4/...
+```
+For EMEA production
+```
+curl -X POST --H 'Content-Type: application/json' --H 'Accept: application/json' --header "Authorization: Bearer {YOUR ACCESS TOKEN}" -d { "name": "Your action name" } https://emea.api.concursolutions.com/travelrequest/v4/...
+```
+For China production
+```
+curl -X POST --H 'Content-Type: application/json' --H 'Accept: application/json' --header "Authorization: Bearer {YOUR ACCESS TOKEN}" -d { "name": "Your action name" } https://cn.api.concurdc.cn/travelrequest/v4/...
+```
+
+
+### <a name="explore-api"></a>Explore the API
+
+#### <a name="prerequisites"></a>Prerequisites
+
+1. [Create a sandbox](https://developer.concur.com/manage-apps/register.html) if you don't already have one.
+2. Read the [Getting Started](https://developer.concur.com/api-reference/authentication/getting-started.html) section of [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+Once you have registered your application, read about the [API endpoints](/api-reference/receipts/endpoints.html), or click the button to download a request collection for Postman.
+
+<a href="https://app.getpostman.com/run-collection/bfe85f4a4e435a161a8a" target="_blank" onclick="ga('send', 'event', 'Postman', 'Click', 'https://app.getpostman.com/run-collection/bfe85f4a4e435a161a8a');">
+  <img src="https://run.pstmn.io/button.svg" alt="Run in Postman">
+</a>
+
+#### <a name="token"></a>Retrieve a User Access Token:
+
+Before making requests to the Requests API, you must [obtain an access token from the Authentication API](https://developer.concur.com/api-reference/authentication/getting-started.html).
+
+The response will include an `access_token` field, which contains your access token. For subsequent calls, you will need to include this access token in the `Authorization` header of your calls. An `id_token` will be also included in the response. In order to retrieve the unique ID for your user, you will have to decode this `id_token` at [jwt.io](https://jwt.io/). You will need this ID in order to post requests.
```

Comment:
You will need this ID in order to post Requests.

Reply:
Here we're talkig about a REST request. I changed it into :